src/cfnlint/data/ExtendedSpecs/all/03_value_types/aws_lambda.json

```diff
@@ -15,6 +15,61 @@
       "NumberMin": 1
     }
   },
+  {
+    "op": "add",
+    "path": "/ValueTypes/AWS::Lambda::Function.MemorySize",
+    "value": {
+      "AllowedValuesDescription": "Number has to be between 128 and 3008 and a multiple of 64.",
+      "AllowedValues": [
```

Comment:
think I'd actually rather see all the available options without having to do any math